server/src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -69,6 +70,9 @@
     /** The recommended number of expired pools to fetch in a single call to listExpiredPools */
     public static final int EXPIRED_POOL_BLOCK_SIZE = 2048;
 
+    /* cf. oracle's maximum; postgres and mysql don't have a limit */
+    public static final int MAX_IN_CLAUSE_VALUES = 1000;
```

Comment:
We already have this constant defined in AbstractHibernateCurator (IN_OPERATOR_BLOCK_SIZE).